### PR TITLE
feat: add managent api

### PR DESCRIPTION
**Pull request by ComradeProgrammer**
feat: add managent api

### CODE REVIEW

**Comment by hsluoyz** [comment]
@ComradeProgrammer plz fix:

image

**Comment by hsluoyz** [comment]
@ComradeProgrammer plz fix:

image

**Comment by hsluoyz** [comment]
@sagilio

File: pkg/client/internal.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	k.Clientset = clientset
+	return nil
+}
+func trim(s string) string {
+	return strings.Trim(s, "\r\n ")
+}
+
+func policyToString(ptype string, param ...string) string {
+	tmp := append([]string{ptype}, param...)
+	return strings.Join(tmp, ",")
```
Comment (sagilio) [inline pkg/client/internal.go]:
```suggestion
	return strings.Join(tmp, ", ")
```

File: pkg/client/policy.go [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+	_, err = k.Clientset.AuthV1().CasbinPolicies(k.namespace).Update(context.TODO(), policyObject, v1.UpdateOptions{})
+
+	if err != nil {
+		return false, err
+	}
+	return true, nil
+}
+
+func (k *K8sGateKeeperClient) SetPolicy(rule []string) (bool, error) {
+	return k.AddNamedPolicies("p", [][]string{rule})
```
Comment (sagilio) [inline pkg/client/policy.go]: It looks like it should call `SetNamedPolicies` func.

**Review by sagilio — CHANGES_REQUESTED** [review]
Most LGTM

**Comment by hsluoyz** [comment]
@sagilio

**Review by sagilio — APPROVED** [review]
LGTM

**Pull request merged**